requirements_all.txt

```diff
@@ -564,7 +564,10 @@ av==13.1.0
 axis==64
 
 # homeassistant.components.fujitsu_fglair
-ayla-iot-unofficial==1.4.7
+ayla-iot-unofficial==1.5.0
+
+# homeassistant.components.nomaiq
+ayla_iot_unofficial==1.5.0
```

Comment:
Let's make sure they follow the same naming. Let's also bump this in a separate PR